earthpy/mask.py

```diff
@@ -86,105 +86,107 @@
 }
 
 
-def make_cloud_mask(mask_arr, vals):
+def _create_mask(mask_arr, vals):
     """Take an input single band mask layer such as a pixel_qa
-    layer for MODIS or Landsat and apply a mask given a range of vals to mask
+    layer for MODIS or Landsat and apply a mask given a range of values to mask.
 
     Parameters
     -----------
     mask_arr : numpy array
-        An array... to open the pixel_qa or mask raster of interest
+        An array of the pixel_qa or mask raster of interest.
 
     vals : list of numbers (int or float)
-        A list of values that represent no data in the provided raster
-        layer (mask_arr)
+        A list of values from the pixel qa layer that will be used to create
```

Comment:
```suggestion
        A list of values from mask_arr (the qa layer) used to create
```

Reply:
Referring to `mask_arr` explicitly here instead of "the pixel qa layer" might be easier for users to see the connection between the `mask_arr` and `vals` arguments

Reply:
perfect!